### PR TITLE
Updated release script in preparation for opensearch-aws-sigv4 release

### PR DESCRIPTION
**Pull request by nhtruong**
Signed-off-by: Theo Truong <theotr@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,14 @@ puts response
     
 ```
 
+## AWS Managed Opensearch with Sigv4
+
+If you have the need to sign every request with AWS Sigv4 to be able to communicate with an Opensearch
```
Comment (dblock) [inline README.md:135]: We usually don't line wrap markdown.

**Review by dblock — DISMISSED** [review]

File: release/build.sh [hunk 2]
```diff
@@ -70,3 +70,16 @@ mv -v opensearch-dsl*.gem $GIT_ROOT/dist/
 
 echo 'List gems to be published: '
 ls -l $GIT_ROOT/dist
+
+
+#####################################################
```
Comment (VachaShah) [inline release/build.sh:75]: Would this section not be before the 'List gems to be published' so that it can also be printed?

Comment (nhtruong) [reply release/build.sh:75]: You're right. Thanks for catching that

**Review by VachaShah — APPROVED** [review]
LGTM

**Comment by nhtruong** [comment]
@dblock would you mind approving this again? Thank you!

**Review by dblock — APPROVED** [review]

**Pull request merged**